iac-extensions/kubernetes/src/main/java/org/sonar/iac/kubernetes/visitors/KubernetesCheckContext.java

```diff
@@ -27,6 +27,8 @@ public interface KubernetesCheckContext extends CheckContext {
 
   void setShouldReportSecondaryInValues(boolean shouldReport);
 
+  void disableLocationShifting();
```

Comment:
I think it's a good idea to have it here, similarly to reporting in values.yaml, so that the rules can decide whether to enable shifting. It could've been more straightforward, if we were able to check rule class without aditional configuration, but context adapter doesn't have it

Reply:
Yeah, unfortunately it's not possible otherwise right now.